testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/HttpClientTest.groovy

```diff
@@ -428,7 +428,7 @@ abstract class HttpClientTest<REQUEST> extends InstrumentationSpecification {
   }
 
   protected String expectedClientSpanName(URI uri, String method) {
-    return method != null ? "HTTP " + method : "HTTP request"
+    return method != null ? method : "HTTP request"
```

Comment:
do we have any tests with null method somehow?
```suggestion
    return method
```

Reply:
Nope, I don't think we do; method is provided by every library we instrument.